packages/zowe-explorer/__tests__/__unit__/uss/ZoweUSSNode.unit.test.ts

```diff
@@ -758,6 +759,38 @@ describe("ZoweUSSNode Unit Tests - Function node.getChildren()", () => {
         globalMocks.withProgress.mockImplementation((progLocation, callback) => {
             return callback();
         });
+        Object.defineProperty(Profiles, "getInstance", {
```

Comment:
I think this mock already exists. Check `createInstanceOfProfile` in `mockCreators/shared.ts`. If you use that you can just change some properties on that object, if you have to for the test, and it will make the definition here much shorter.

Reply:
I need help. I tried this but its wrong 😢 

```
testCombinedProfile: createValidIProfile()
....
newMocks.mockProfilesInstance = createInstanceOfProfile(newMocks.testCombinedProfile);
newMocks.mockProfilesInstance.getCombinedProfile = jest.fn(() => {
            return [{ name: newMocks.testCombinedProfile.name, profile: newMocks.testCombinedProfile }];
        });
```

Reply:
Same result if I use this `newMocks.mockProfilesInstance.getCombinedProfile.mockReturnValue(newMocks.testCombinedProfile);`

Reply:
Checking...

Reply:
Ok, maybe try:

_newMocks.mockProfilesInstance.getCombinedProfile._**mockResolvedValue**_(newMocks.testCombinedProfile);_

It seems to work for me